### ISSUE TITLE
Shiva's Guard doesn't apply AS reduction during Phoenix's Supernova

### ISSUE THREAD

**Issue by icecoldfeathers** [ValveSoftware/Dota2-Gameplay#6395]
### Item name

Shiva's Guard

### Description

Shiva's Guard doesn't apply AS reduction during Supernova. However, Radiance's DPS aura and miss chance works. 

### Example Match ID and Timestamp

6947668557 - 19:05

**Comment by shaileshgarg** [comment]
Before Ulti
![20230105222120_1](https://user-images.githubusercontent.com/26450759/210835901-ad3508b7-92c2-4b4b-ae69-dc5fc073621c.jpg)

After Ulti
![20230105222131_1](https://user-images.githubusercontent.com/26450759/210835906-1eeb1db8-5084-4f6e-8f08-69c20c8dbc55.jpg)


Phoenix has Shiva's Guard,, seems to be working fine in Demo.

**Comment by shaileshgarg** [comment]
Yeah, LC is not getting the AC debuff Icon when Phoenix Ultis, but the actual Attack Speed is reduced. I have added the screenshot above. The AS goes from 181 -> 136

**Comment by shaileshgarg** [comment]
https://dota2.fandom.com/wiki/Version_6.82, AC and Shiva's not showing debuff, but applying when enemy is not visible is intended. 

During Phoenix Ulti the actual hero is hidden, so the debuff icons are not shown but the actual reductions are visible in the HUD.
